akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Protocol.scala

```diff
@@ -17,6 +17,8 @@ object Http2Protocol {
   val InitialConnectionLevelWindow = 65535
   val InitialStreamLevelWindow = 65535
 
+  val NoStreamId = 0
```

Comment:
ok, since `1` is the first ID used by the actual streams, perhaps add such comment?